test/Microsoft.AspNet.Http.Abstractions.Tests/project.json

```diff
@@ -6,13 +6,26 @@
   "dependencies": {
     "Microsoft.AspNet.Http": "1.0.0-*",
     "Microsoft.AspNet.Testing": "1.0.0-*",
-    "xunit.runner.aspnet": "2.0.0-aspnet-*"
+    "xunit": "2.1.0"
   },
+  "frameworks": {
+    "dnx451": {
+     "frameworkAssemblies": {
+        "System.Runtime": "",
+        "System.Threading.Tasks": ""
```

Comment:
Because `xunit.runner.aspnet` (or one of its dependecies) was bringing them before; `xunit` doesn't.